docs/source/developer-guidelines/testing.md

```diff
@@ -22,11 +22,13 @@ environment variable.
 
 This is specially useful in CI while developing add-ons, so you can pass an specific configuration that deals with the addon config properly.
 
-## Add addons for testing purposes
+## Add addons via environment variable (for testing purposes)
 
-Quite often, you need different configurations and enabling components for testing purposes. You can use the `RAZZLE_TESTING_ADDONS` environment variable to define them. These addons do not have the capacity to install dependencies (as opposed to properly defined addons), but all the rest of the addon features work as expected.
+Sometimes, you need to enable different configurations and enable optional components (eg. for testing purposes). You can use the `ADDONS` environment variable to define them.
 
-    RAZZLE_TESTING_ADDONS=test-addon,test-addon2 yarn start
+    ADDONS=test-addon,test-addon2 yarn start
```

Comment:
Let's explicitly specify the syntax highlighting.

````suggestion
```bash
ADDONS=test-addon,test-addon2 yarn start
```
````